unpackaged/config/installer/profiles/Admin.profile

```diff
@@ -5,4 +5,8 @@
         <default>true</default>
         <visible>true</visible>
     </applicationVisibilities>
+    <tabVisibilities>
+        <tab>hed__Application__c</tab>
+        <visibility>DefaultOn</visibility>
```

Comment:
I still had to click the pencil and add it to the tabs, should we document that?

Reply:
@skosgi1,  we can unless @spinkelman @jacebryan @lifewithryan thinks we should have it on the actual tab list and not have to edit the tabs to add it.

Reply:
FYI @juliesnow

Reply:
FYI @juliesnow

Reply:
@lifewithryan @juliesnow @JasonFitzy If the intent is to make the K-12 kit have the applications tab in its tab list, we'll need to add it to the managed package. In that case, everyone that has access to the applications tab will see it there.

Currently, the behavior is that if one is in the K-12 kit and selects the applications tab from the app launcher, they will get to the desired page layout.

Reply:
It was my impression that we didn't necessarily want it on their default list of tabs, but should be able to access it via the App Launcher/Waffle.

Reply:
@skosgi1 @MatthewBlanski @lifewithryan , the original request came from @spinkelman that Application's tab should be enabled by default but it was not requested that it should be defaulted in the actual tab list. Defaulting in the tab list could not be 100% necessary for all our customers since they might not be heavily using that.

Reply:
Kim is correct. We just need the Application object to be "Default on" for the System Administrator user. The actual object doesn't need to be listed as a top nav tab view for the users. 

To have the object visible in the K12 app is called "Tab Settings" so that's where the confusion is coming from.